src/main/java/com/liferay/faces/util/el/internal/UtilELResolver.java

```diff
@@ -39,17 +42,16 @@
 	// serialVersionUID
 	private static final long serialVersionUID = 4993137243887595902L;
 
-	static {
+	// Private Data Members
+	private final I18nMap i18nMap = new I18nMap();
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.